Dockerfile

```diff
@@ -1,16 +1,19 @@
-FROM python:3.12-alpine@sha256:5049c050bdc68575a10bcb1885baa0689b6c15152d8a56a7e399fb49f783bf98 AS base
+# Always use the latest image
+# hadolint ignore=DL3007
+FROM cgr.dev/chainguard/wolfi-base:latest AS base
 ENV PYTHONUNBUFFERED=1
 
 FROM base AS builder
 # we want always the latest version of fetched apk packages
 # hadolint ignore=DL3018
-RUN apk add --no-cache build-base libressl-dev musl-dev libffi-dev && \
+RUN apk add --no-cache build-base openssl-dev glibc-dev posix-libc-utils libffi-dev \
+    python-3.12 python3-dev py3.12-pip && \
```

Comment:
Do we need `python3-dev`? Maybe other package are also not needed? But I could be wrong.